integration_tests/examples/easy_test.rs

```diff
@@ -0,0 +1,225 @@
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_7/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_7_PORT,
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register output request.");
+    examples::post(
+        "output/output_1/register",
+        &json!({
+            "type": "rtp_stream",
+            "ip": IP,
+            "port": OUTPUT_VIDEO_PORT,
+            "video": {
+                "resolution": {
+                    "width": VIDEO_RESOLUTION.width,
+                    "height": VIDEO_RESOLUTION.height,
+                },
+                "encoder": {
+                    "type": "ffmpeg_h264",
+                    "preset": "fast"
+                },
+                "initial": {
+                    "root": {
+                        "type": "tiles",
+                        "children": [
+                            {
+                                "type": "input_stream",
+                                "input_id": "input_1"
+                            },
+                            {
+                                "type": "input_stream",
+                                "input_id": "input_3"
+                            },
+                            {
+                                "type": "input_stream",
+                                "input_id": "input_5"
+                            },
+                            {
+                                "type": "input_stream",
+                                "input_id": "input_6"
+                            },
+                            {
+                                "type": "input_stream",
+                                "input_id": "input_7"
+                            }
+                        ]
+                    }
+                },
+                "resolution": { "width": VIDEO_RESOLUTION.width, "height": VIDEO_RESOLUTION.height },
+            }
+        }),
+    )?;
+
+    info!("[example] Send register output request.");
+    examples::post(
+        "output/output_2/register",
+        &json!({
+            "type": "rtp_stream",
+            "ip": IP,
+            "port": OUTPUT_AUDIO_PORT,
+            "audio": {
+                "initial": {
+                    "inputs": [
+                        {"input_id": "input_2"},
+                        {"input_id": "input_4"}
+                    ]
+                },
+                "encoder": {
+                    "type": "opus",
+                    "channels": "stereo",
+                }
+            }
+        }),
+    )?;
+
+    std::thread::sleep(Duration::from_millis(500));
+
+    info!("[example] Start pipeline");
+    examples::post("start", &json!({}))?;
+
+    ff_stream_sample(
```

Comment:
I would prefer `ffmpeg` over `ff`. Also maybe to unify naming a bit, maybe
- `ffmpeg_stream_send` / `ffplay_stream_receive`
- `start_ffmpeg_send` / `start_ffplay_receive` (start implies that it is not blocking)